src/core/Entry.cpp

```diff
@@ -916,11 +917,83 @@ QString Entry::resolvePlaceholderRecursive(const QString& placeholder, int maxDe
     }
     case PlaceholderType::Reference:
         return resolveReferencePlaceholderRecursive(placeholder, maxDepth);
+    case PlaceholderType::DateTimeSimple:
+    case PlaceholderType::DateTimeYear:
+    case PlaceholderType::DateTimeMonth:
+    case PlaceholderType::DateTimeDay:
+    case PlaceholderType::DateTimeHour:
+    case PlaceholderType::DateTimeMinute:
+    case PlaceholderType::DateTimeSecond:
+    case PlaceholderType::DateTimeUtcSimple:
+    case PlaceholderType::DateTimeUtcYear:
+    case PlaceholderType::DateTimeUtcMonth:
+    case PlaceholderType::DateTimeUtcDay:
+    case PlaceholderType::DateTimeUtcHour:
+    case PlaceholderType::DateTimeUtcMinute:
+    case PlaceholderType::DateTimeUtcSecond:
+        return resolveMultiplePlaceholdersRecursive(resolveDateTimePlaceholder(typeOfPlaceholder), maxDepth - 1);
     }
 
     return placeholder;
 }
 
+QString Entry::resolveDateTimePlaceholder(Entry::PlaceholderType placeholderType) const
+{
+    time_t time = std::time(NULL);
```

Comment:
Use nullptr, not NULL.

Reply:
Also all time must be derived used the `Clock` class.

Reply:
Thanks for the hint `Clock`. I've changed that.
Usage of `NULL` / `nullptr` is obsolete with the current changes :)